src/docs/Components/Dividers/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Dividers'
+order: 1
+description: 'Dividers are thin lines used to insert breaks into your layouts and group similar content.'
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> Dividers are thin lines used to insert breaks into your layouts and group similar content.
+
+--Preview--
+
+## Variations
+
+There are four different variations depending on your page needs. Every color variation has a vertical or an horizontal version to suit your pages design.
+
+### Lightest
+
+This lightest version must be used on dark background only to be visible.
+
+### Light
+
+We provide a light version of the dividers but you must be aware that this version is not AA accessible. This version must be used for dividers that are not mandatory to understand the content of your page.
+
+<Highlight theme="warning" title="Be aware">
+
+  This version will only be visible on high end screens and won't be seen by people who have sight troubles.
+
+</Highlight>
+
+### Default
+
+This is the version you should use in your layouts. This default version is AAA accessible and will be visible by every users.
+
+### Dark
```

Comment:
Could the variations be categorized from those that should be used most often to those that should be used least often? Maybe the "default" first and then the others (Dark, Light, Lightest)?

Reply:
I agree with you @tiloyi

Reply:
@tiloyi yes, good idea 👍